packages/mui-material/src/TextField/TextField.test.js

```diff
@@ -213,7 +213,7 @@ describe('<TextField />', () => {
       );
 
       const input = container.querySelector('input[aria-hidden]');
-      expect(input).not.to.have.attribute('id');
+      expect(input).to.have.attribute('id');
       expect(input).not.to.have.attribute('aria-describedby');
```

Comment:
If we're changing the logic to include the id, perhaps we should also add the label-id to the aria-describedby as well?

Reply:
Nevermind, I just re-read and realised you used the htmlFor attribute on the label instead

Though this *may* actually cause an accessibility issue when the screenreader finds the label but can't find the associated input (because it's aria-hidden)

Reply:
So what do you suggest I do?

Reply:
@Owenleo778 I completely understand your point. I'm not very experienced with accessibility either. I did check the Chrome issues tab, and it seems that there are no reported issues related to the changes made. If you have any specific recommendations or resources on accessibility best practices, I'd be more than happy to learn and implement them.